index.html

```diff
@@ -10,7 +10,7 @@
             href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700;800&display=swap"
             rel="stylesheet"
         />
-        <title>QBus - Bus Ticket Booking</title>
+        <title>Bustle - journeys redefined</title>
```

Comment:
I think "Journeys" will be better

Reply:
Resolved in ed36de56ec2a66f79e07fbba35626656834f4b98